files/es/web/html/element/input/range/index.md

```diff
@@ -199,44 +199,49 @@ Captura de pantalla
 
 ### Cambiar la orientación
 
+### Crear controles de rango vertical
+
 Por defecto, si un navegador renderiza un input range, lo mostrará como un "slider" (deslizador) que se desliza hacia la izquierda y hacia la derecha. By default, if a browser renders a range input as a slider, it will render it so that the knob slides left and right. Sin embargo puedes cambiar esto fácilmente para que se deslice hacia arriba y hacia abajo simplemente usando CSS
 
 > **Nota:** Esto aún no está implementado por los principales navegadores. This is not actually implemented yet by any of the major browsers. See Firefox [Error 981916 en Firefox](https://bugzil.la/981916), [Chrome bug 341071](https://bugs.chromium.org/p/chromium/issues/detail?id=341071).
 
+#### Control de rango horizontal
+
 Si tenemos el siguiente control range:
 
 ```html
 <input type="range" id="volume" min="0" max="11" value="7" step="1" />
 ```
 
-{{EmbedLiveSample("Orientation_sample1", 200, 200, "orientation_sample1.png")}}
+{{EmbedLiveSample("Control de rango horizontal", 200, 40)}}
 
-Dicho control se muestra en horizontal (al menos en los principales navegadores, o otros puede variar). Presentarlo en vertical es tan simple como añadir CSS para cambiar las dimensiones del control, de la siguiente manera:
+Dicho control se muestra en horizontal (al menos en los principales navegadores, o otros puede variar).
 
-#### CSS
+#### Usando la propiedad de appearance
 
-```css
-#volume {
-  height: 150px;
-  width: 50px;
-}
-```
+La propiedad {{cssxref('appearance')}} tiene un valor no estándar de `slider-vertical` que, bueno, hace que los controles deslizantes sean verticales.
 
-#### HTML
+Usamos el mismo HTML que en los ejemplos anteriores:
 
 ```html
-<input type="range" id="volume" min="0" max="11" value="7" step="1" />
+<input type="range" min="0" max="11" value="7" step="1" />
 ```
 
-#### Result
+Nos dirigimos solo a las entradas con un tipo de rango:
+
+```css
+input[type="range"] {
+  appearance: slider-vertical;
+}
+```
 
-{{EmbedLiveSample("Orientation_sample2", 200, 200, "orientation_sample2.png")}}
+{{EmbedLiveSample("Usando la propiedad de appearance", 200, 200)}}
 
-**En la actualidad, ninguno de los navegadores principales soporta la creación de inputs range usando este CSS, incluso a perar de que la especificación recomienda que lo hagan.**
+#### Control de rango vertical
```

Comment:
The following content has not been updated (We can update this content in the future)